llama_index/callbacks/open_inference_callback.py

```diff
@@ -0,0 +1,315 @@
+"""
+Callback handler for retrieval-augmented generation data in OpenInference
```

Comment:
I'm not sure I'd mention RAG here? It's meant to be more generic if possible

Reply:
Addressed in [f968ba9](https://github.com/jerryjliu/llama_index/pull/6998/commits/f968ba9d09d00cdb3b3662d37667b81c74ccb323)